lib/validators/color_list_validator.rb

```diff
@@ -0,0 +1,14 @@
+class ColorListValidator
+  def initialize(opts = {})
+    @opts = opts
+  end
+  
+  def valid_value?(value)
+    hex_regex = /\A[0-9a-fA-F]{6}\z/
+    value.split("|").all? {|c| c =~ hex_regex}
```

Comment:
Missing some whitespaces

```suggestion
    value.split("|").all? { |c| c =~ hex_regex }
```